src/renate/benchmark/experimentation.py

```diff
@@ -252,7 +252,7 @@ def _execute_experiment_job_locally(
     See renate.benchmark.experimentation.execute_experiment_job for more details.
     """
     logger.info("Start experiment.")
-    seed_everything(seed)
+    seed_everything(seed, True)
```

Comment:
not super sure but according to my interpretation of https://pytorch-lightning.readthedocs.io/en/1.6.5/api/pytorch_lightning.utilities.seed.html#pytorch_lightning.utilities.seed.seed_everything it will seed the workers

Reply:
https://pytorch.org/docs/stable/notes/randomness.html#dataloader